zproject/urls.py

```diff
@@ -142,6 +142,14 @@
     # password/pair and returns an API key.
     url(r'^api/v1/fetch_api_key$',          'api_fetch_api_key'),
 
+    # This is for the signing in through the devAuthBackEnd.
+    url(r'^api/v1/dev_android_direct_login$',  'api_dev_android_direct_login'),
+
```

Comment:
Can you remove this blank line?  It'd be great to group these two blocks together.